src/client/windshaft.js

```diff
@@ -160,7 +165,7 @@ export default class Windshaft {
         };
         if (metadata) {
             if (metadata.featureCount >= 0) {
-                choices.backendFilters = metadata.featureCount > MIN_FILTERING;
+                choices.backendFilters = metadata.featureCount > MIN_FILTERING || !metadata.backendFiltersApplied;
```

Comment:
I don't understand this. Why not just `choices.backendFilters = metadata.backendFiltersApplied;` ?